z_infra_provisioning/cloud_infra_center/ocp_upi/add-new-compute-node.yaml

```diff
@@ -248,34 +252,30 @@
         cluster_domain_name: "{{ hostvars['localhost']['cluster_name'] }}.{{ hostvars['localhost']['base_domain'] }}"
       with_items: "{{ hostvars['localhost']['workers_list'] }}" 
       when:
-        - update_bastion is defined
-        - update_bastion == "true"
-    
+        - use_internal_bastion == True
```

Comment:
The inventory is using `true` for parameter use_internal_bastion, pls be consistent.
- use_internal_bastion == true